serde-generate/src/test_utils.rs

```diff
@@ -337,9 +337,9 @@ pub enum Runtime {
     Bincode,
 }
 
-impl std::convert::Into<Encoding> for Runtime {
-    fn into(self) -> Encoding {
-        match self {
+impl std::convert::From<Runtime> for Encoding {
```

Comment:
Was this just wrong before?

Reply:
No, implementing `std::convert::From<Bla> for Foo` is just better practice than implementing `std::convert::Into<Foo> for Bla` (as the latter can be deduced from the former)